backend/controllers/LocationsController.ts

```diff
@@ -44,23 +43,21 @@ const LocationsController = {
           locations.length < 50 ? locations.length : 50
         );
       }
+      if (!locations) {
+        return res.status(404).json({ message: req.t('locations_not_found') });
```

Comment:
req.t  - I don't understand what this is - can we change t ->  translate?

Reply:
the "t" function itself comes form req(request object),so it must be this way